pkg/auth/authenticator/password/ldappassword/util/server.go

```diff
@@ -0,0 +1,73 @@
+package util
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net"
+
+	"github.com/go-ldap/ldap"
+)
+
+// LDAPServerConfig holds information for connecting to an LDAP server
+type LDAPServerConfig struct {
```

Comment:
I see. Your original comment made it sound like you wanted the only input to be the `LDAPURL`. That's reasonable.